scripts/e2e.mjs

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env node
+import { execSync, spawn } from 'child_process';
+import * as fs from 'fs';
+import * as path from 'path';
+import { argv, env } from 'process';
+import { fileURLToPath } from 'node:url';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+if (argv.length < 3) {
+  console.error(`Usage: ${path.basename(__filename)} <platform>`);
+  process.exit(1);
+}
+
+const platform = argv[2];
+if (platform !== 'ios' && platform !== 'android') {
+  console.error(`Unsupported platform: ${platform}`);
+  process.exit(1);
+}
+
+var actions = ['create', 'build', 'test'];
+if (argv.length >= 4) {
+  var newActions = [];
+  for (let index = 0; index < actions.length; index++) {
+    const action = actions[index];
+    if (argv.includes(`--${action}`) || argv.includes(`-${action}`) || argv.includes(action)) {
+      newActions.push(action);
+    }
+  }
+  actions = newActions;
+}
+console.log(`Performing actions: ${actions}`);
+
+env.SENTRY_DISABLE_AUTO_UPLOAD = 'true'
```

Comment:
```suggestion
if (env.SENTRY_DISABLE_AUTO_UPLOAD === undefined) {
  // Auto upload to prod made the CI flaky
  // This can be removed in the future or when mocked server is added
  env.SENTRY_DISABLE_AUTO_UPLOAD = 'true'
}
```